test/Benchmarks/src/Sieve/Sieve_Ascribed.enso

```diff
@@ -16,9 +16,9 @@ type Filter
     Item n:Integer next:Filter
     Head first:Filter set_last:Filter->Nothing
 
-    acceptAndAdd : Integer
-    acceptAndAdd self n =
-        iterate f = case f of
+    acceptAndAdd : Integer -> Filter
```

Comment:
There are supposed to be some changes in https://github.com/enso-org/enso/pull/7882/files#diff-f318804809286bb567f4076547fa2ed5455b0662960e4c9c295287582b5bc9daR30 because of interaction with JavaScript.